microsoftgraph/microsoft-graph-explorer-v4
# Task: Order permissions according to privilege

## Overview
Implements #1933 

### Demo
![image](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/assets/45680252/7374b24a-f86f-4b8f-a1bc-12920b700fa4)



### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Click on the Modify permissions tab
Notice that the permissions are ordered with the least privileged permission appearing first on the list
